pkg/coremodel/dashboard/coremodel.cue

```diff
@@ -24,6 +24,9 @@ seqs: [
 				// Description of dashboard.
 				description?: string
 
+				// Version of the current dashboard data 
+				revision: int64 @grafanamaturity(NeedsExpertReview)
```

Comment:
oooh this is a weird on.  Looks like it is specific to dashboards supplied by plugins https://github.com/grafana/grafana/blob/main/public/app/types/plugins.ts#L15